src/common/components/ErrorHandling/WithErrorBoundary.tsx

```diff
@@ -4,26 +4,33 @@ import { ErrorBoundary, FallbackProps } from 'react-error-boundary';
 import { ErrorState } from '../ui';
 import { useErrorMonitor } from './ErrorMonitorContext';
 
-const getErrorFallbackComponent = ({ title }: WithErrorBoundaryProps) => {
+const getErrorFallbackComponent = ({
+  title,
+  content = 'There was an internal error',
+  showReloadAction = true,
+}: WithErrorBoundaryProps) => {
   const FallbackComponent = ({ resetErrorBoundary }: FallbackProps) => {
-    const content = 'There was an internal error';
-    const primaryAction = [
+    const primaryAction = showReloadAction ? (
       <Button
         onClick={resetErrorBoundary}
         variant={ButtonVariant.link}
         isInline
         key="reset-error-boundary"
       >
         try again
-      </Button>,
-    ];
+      </Button>
+    ) : (
+      <></>
```

Comment:
If it is supposed to render nothing...
```suggestion
      null
```